src/components/Menu/MenuItem.tsx

```diff
@@ -43,11 +51,38 @@ export function MenuItem({
   const { menuOpen, toggleMenuOpen } = useContext(LayoutContext);
   const asPathWithoutHash = usePathWithoutHash();
   const [open, setOpen] = useState(false);
+  const ref = useRef(null);
   const children = useMemo(
     () => (hideChildren ? [] : pageNode.children),
     [hideChildren, pageNode.children]
   );
+
+  const setSelectability = () => {
+    const current = ref.current;
+    const items = current?.parentElement?.nextSibling?.children;
+    if (items) {
+      for (const item of items) {
+        const links = item.getElementsByTagName('a');
+        const buttons = item.getElementsByTagName('button');
+        if (links[0].getAttribute('tabIndex') == 0) {
+          for (const link of links) {
+            link.setAttribute('tabIndex', -1);
+          }
+          for (const button of buttons) {
+            button.setAttribute('tabIndex', -1);
+          }
+        } else if (-links[0].getAttribute('tabIndex')) {
+          links[0]?.setAttribute('tabIndex', 0);
```

Comment:
It looks like the tabIndex can get out of sync with the display by clicking on the button and the link.  I wonder if there is another attribute that could be used to determine if this should be tabbable.  Something like tracking the open state in the parent and using that isOpen state to determine the tabIndex of the menu items.

https://github.com/user-attachments/assets/ab48543f-2031-4d61-a027-1b57903f16c2